### DISCUSSION TITLE
Set a user to operator via docker-compose

### DISCUSSION THREAD

**Discussion by Daimonion1980** [itzg/docker-minecraft-server#2962] [Q&A] [answered]
Hey, is it possible to set a user (its userid) to op via docker-compose?

I have a java version running via docker-compose in portainer (itzg together with mc-backup) and want to set one user as op so that he is able to ban users who are not following the rules. 
Alternatively is it possible to set this via attaching to the containers shell and set this manually? Will this setting then persist a reboot of the container?

**Comment by itzg** [comment, marked as answer]
https://docker-minecraft-server.readthedocs.io/en/latest/configuration/server-properties/#opadministrator-players

**Reply by Daimonion1980** [reply]
Thank you very much!
